db/models/feedback.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Feedback = db.define('feedback', {
```

Comment:
From people, for us. In the google doc there are already one or two points of feedback